### PR TITLE
Add Osiris and False

### PR DESCRIPTION
**Pull request by ghost**
Tested and works :DDD

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -1043,6 +1067,35 @@ let BattleMovedex = {
...
+		category: "Physical",
+		desc: "",
+		shortDesc: "",
+		id: "nightmarch",
+		name: "Night March",
+		pp: 5,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Fire Lash", target);
```
Comment (ghost) [inline mods/seasonal/moves.js]: Need animation from him, so hold on until i change this

**Comment by ghost** [comment]
erm, how to fix ts?

**Comment by HoeenCoder** [comment]
Also handle conflicts

**Comment by ghost** [comment]
OK review away!

File: mods/seasonal/moves.js (hunk 1 above)
Comment (ghost) [reply mods/seasonal/moves.js]: ☆Osiris: Nightmare + Moongeist Beam + Stomping Tantrum
☆Lycanium Z: Nightmare then moongeist meam then stomping tantrum (I cant do them at the same time)
☆Lycanium Z: ?
☆Lycanium Z: uh
☆Osiris: Yup!
☆Lycanium Z: Ok
☆Lycanium Z: Thanks for letting me know

**Comment by ghost** [comment]
@HoeenCoder ready to be merged

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -468,6 +468,30 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Normal",
 	},
+	// False
+	frck: {
+		accuracy: true,
+		basePower: 1000,
+		category: "Status",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js]: Status? shouldn't it be physical instead

**Pull request merged**